activesupport/lib/active_support/inflector/methods.rb

```diff
@@ -195,12 +197,7 @@ def dasherize(underscored_word)
     #
     # See also #deconstantize.
     def demodulize(path)
-      path = path.to_s
-      if i = path.rindex('::')
-        path[(i+2)..-1]
-      else
-        path
-      end
+      (i = path.rindex('::')) ? path[i + 2..-1] : path
```

Comment:
Ternary is harder to visually parse. `i + 2..-1` spacing makes it look like we're adding 1 to a Range.